package.json

```diff
@@ -97,10 +101,8 @@
     "standard-version": "^7.0.0",
     "trash-cli": "^3.0.0",
     "ts-jest": "^24.1.0",
-    "tslint": "^5.10.0",
-    "tslint-config-standard": "^8.0.1",
     "typedoc": "^0.15.0",
-    "typescript": "~3.6.0"
+    "typescript": "4"
```

Comment:
Why is this just "4"? Shouldnt it be "~4.0" (or "4.0.x")
Same question about eslint.

I dont know how those get interpreted in semver for updates

Reply:
No reason, good catch